### PR TITLE
fix build migrate to Go modules

### PR DESCRIPTION
**Pull request by szuecs**
fix build that was broken while reafctoring the loadbalancer implementation

Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
 GOX_OUTPUT ?= $(BUILD_DIR)/{{.Dir}}_{{.OS}}_{{.Arch}}
+GOMODULE   ?= GO111MODULE=on
 
 default: install test build
 
 install:
-	go get -t -v ./...
-	go get github.com/mitchellh/gox
+	$(GOMODULE) go get -t -v ./...
+	$(GOMODULE) go get github.com/mitchellh/gox
```
Comment (mikkeloscar) [inline Makefile]: This step is  not needed with go modules :)

File: Makefile [hunk 2]
```diff
@@ -1,21 +1,22 @@
...
 GOX_ARCH   ?= darwin/386 darwin/amd64 linux/amd64 linux/386
 GOX_OUTPUT ?= $(BUILD_DIR)/{{.Dir}}_{{.OS}}_{{.Arch}}
+GOMODULE   ?= GO111MODULE=on
 
 default: install test build
 
 install:
-	go get -t -v ./...
-	go get github.com/mitchellh/gox
+	$(GOMODULE) go get -t -v ./...
```
Comment (mikkeloscar) [inline Makefile:9]: This step is not needed with go modules :)

**Comment by rbarilani** [comment]
@szuecs Wow thanks Sandor!

File: Makefile (hunk 2 above)
Comment (rbarilani) [reply Makefile:9]: @mikkeloscar I will remove that with a sub-sequent PR

**Pull request merged**